package.json

```diff
@@ -5,12 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test": "grunt test",
-    "start": "node ./sandbox/server.js",
+    "start": "nodemon ./sandbox/server.js",
```

Comment:
Why there are non-related script changes here?

Reply:
sorry,  I pushed some useless code, I will handle it.